silkshopy-ecommerce/src/pages/ProductListing/ProductListing.jsx

```diff
@@ -28,7 +27,7 @@ const ProductListing = () => {
           <h2 className="product-main-heading margin-bottom">Products</h2>
 
           <div className="product-cards">
-            {product && sortedData.map((item) => ( <Card key={item.id} productData = {item} /> ))}
+            {product && sortedData.map((item) => ( <Card key={item.id} productData = {item}  /> ))}
```

Comment:
You can use ternary operator instead of AND operator to avoid error in edge cases